samples/graph-chat-lifecycle/csharp/ChatLifecycle/Views/Home/Index.cshtml

```diff
@@ -30,7 +30,8 @@
 
 <script>
   
-    microsoftTeams.app.initialize();
+    microsoftTeams.app.initialize().then(() => {
```

Comment:
please try once with await microsoftTeams.app.initialize()

Reply:
Please update at all places if above code works

Reply:
Hi @Prithvi-MSFT 

I have tried, as per your PR comments, to **await microsoftTeams.app.initialize()**, but i am getting this error.


![image](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/117716554/7bb2ac8e-8163-41a8-beed-de2ac4d06496)

![image](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/117716554/337e302e-0502-4838-b143-797c57bc7184)